docs/data-management-on-carbondata.md

```diff
@@ -126,20 +126,20 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
 
    - **Local Dictionary Configuration**
      
-     Local Dictionary is generated only for no-dictionary string/varchar datatype columns. It helps in:
+     Local Dictionary is generated only for string/varchar datatype columns which are not included in dictionary include. It helps in:
```

Comment:
Please add one Note  and list which data type don't support.

Reply:
Add a small sentence on what local dictionary means